configs/starfive/visionfive2/kernel/extra

```diff
@@ -0,0 +1,2 @@
+CONFIG_CRYPTO_SALSA20=n
```

Comment:
Also minor, but it's more "standard" to use the following syntax to unset the variable:

```
# CONFIG_CRYPTO_SALSA20 is not set
# CONFIG_CRYPTO_SM4 is not set
```